client/src/modules/stock/import/stockImport.js

```diff
@@ -0,0 +1,108 @@
+angular.module('bhima.controllers')
+  .controller('StockImportController', StockImportController);
+
+// dependencies injections
+StockImportController.$inject = [
+  'DepotService', 'InventoryService', 'NotifyService', 'SessionService', 'util',
+  'bhConstants', 'ReceiptModal', 'StockFormService', 'StockService',
+  'StockModalService', 'uiGridConstants', 'appcache', 'Upload', '$state',
+];
+
+/**
+ * @class StockImportController
+ *
+ * @description
+ * This module helps to import stock from a file
+ */
+function StockImportController(
+  Depots, Inventory, Notify, Session, util, bhConstants, ReceiptModal, StockForm,
```

Comment:
It looks like there is a lot of imports here that you never use.  Can you clean this up?